.github/workflows/cross-compile.yml

```diff
@@ -3,13 +3,14 @@ jobs:
   crosscompile:
     strategy:
       matrix:
-        go: [ "1.14.x", "1.15.x" ]
+        go: [ "1.14.x", "1.15.x", "1.16.0-beta1" ]
     runs-on: ubuntu-latest
     name: "Cross Compilation (Go ${{matrix.go}})"
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
+          stable: contains(${{ matrix.go }}, "beta") || contains(${{ matrix.go }}, "rc")
```

Comment:
should this not be "unstable" or "prerelease"?

Reply:
Not sure what you mean, but I think I go the condition here wrong (need to negate it).

Reply:
Yeah, that's what I mean. The logic you have is the opposite of stable, so I was suggesting names.

Reply:
`stable` is defined by the setup-go, so we can't change that. I agree with you that it would be better if a boolean option with a default would default to `false`, but maybe that's a too Go-y way of thinking.